### PR TITLE
Add --omit-hash CLI argument to omit hashes from output filenames.

### LINKED ISSUE #267: Can we simplify the build output?
**Enhancement**

Package Version: 5.0.2

I'm trying to integrate dojo into a simple Django app. Ideally I would like to be able to simply ask dojo to dump it's built JS and CSS files into a static files folder without any hashes in the file name.

I don't like the idea of dropping the dojo cli tooling completely to manage this, and I don't like the idea of working around the unpredictable file names in other specific build steps.

I have two questions:

* Would you be interested in a PR adding a config/cmdline option for not including a hash in the file names (as is done currently for dev builds)?
  * e.g. `--no-hash`
  * I would like to manage cache control in the same way as the rest of the project.
* Would you be interested in a PR adding a config/cmdline option for minimising build output (no source maps, no instrumentation, just the run-time required files)?
  * e.g. `--minimal`

### PR DESCRIPTION
**Pull request by willstott101**
**Type:** feature

The following has been addressed in the PR:

* [x] There is a related issue
* [x] All code has been formatted with [`prettier`](https://prettier.io/)
* [ ] Unit or Functional tests are included in the PR

**Description:**

Resolves #267 

I'm happy to add some tests but honestly I'm bit overwhelmed by the test code. I assume I would be best adding something like https://www.chaijs.com/plugins/chai-fs/ ? I'm probably missing something. _Existing tests do pass though._

### CODE REVIEW

File: src/dist.config.ts [hunk 1]
```diff
@@ -128,7 +128,7 @@ function webpackConfig(args: any): webpack.Configuration {
 		...output,
 		path: outputPath,
 		chunkFilename: '[name].[chunkhash].bundle.js',
```
Comment (matt-gadd) [inline src/dist.config.ts]: do you want to drop the `chunkhash` from here too?

**Comment by matt-gadd** [comment]
@willstott101 don't worry about tests, we mostly only have the integration `test-app` which is more of a smoke test than anything 👍

File: src/dist.config.ts (hunk 1 above)
Comment (willstott101) [reply src/dist.config.ts]: Yes I do! I was testing with --single-bundle and didn't really know what `chunkFilename` was.

**Comment by matt-gadd** [comment]
@willstott101 looks good to me, gonna land this and backport to the 5.x branch and we'll publish a release today or tomorrow. thanks for the contribution!! 🎉

**Comment by willstott101** [comment]
Great, thank you!

**Pull request merged**